samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/AppManifest/manifest.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
```

Comment:
Let's start using latest manifest for all new samples.

Reply:
Fixed